client/MANIFEST

```diff
@@ -7,6 +7,7 @@
 ./pdo/client/builder/shell.py
 ./pdo/client/builder/state.py
 ./pdo/client/commands/__init__.py
+./pdo/client/commands/collection.py
```

Comment:
How about adding a Entrypoint to setup.py ? Is there any reason why you wouldnt want to expose the  'do_collection' commands ?

Reply:
good suggestion. will do

Reply:
done